lib/model/unittest/CMetricModelTest.cc

```diff
@@ -2329,7 +2335,7 @@ BOOST_FIXTURE_TEST_CASE(testIgnoreSamplingGivenDetectionRules, CTestFixture) {
     // Check the last value times of the underlying models are the same
     // const maths::CUnivariateTimeSeriesModel *timeSeriesModel =
     //     dynamic_cast<const maths::CUnivariateTimeSeriesModel*>(modelWithSkipView->model(model_t::E_IndividualMeanByPerson, 0));
-    // BOOST_TEST_REQUIRE(timeSeriesModel != 0);
+    // BOOST_TEST_REQUIRE(timeSeriesModel != nullptr);
```

Comment:
This same change can be made on line 1319 of `lib/model/unittest/CMetricPopulationModelTest.cc`